target_chains/ethereum/contracts/contracts/executor/Executor.sol

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.12;
+
+import "../pyth/PythGovernanceInstructions.sol";
```

Comment:
I recommend removing this dependency so in the future we can use this alone on Pyth itself.